default/stringtables/en.txt

```diff
@@ -1915,6 +1915,9 @@ Automatically quits once any turns specified by --auto-advance-n-turns are compl
 OPTIONS_DB_BINARY_SERIALIZATION
 Use Binary serialization for saving games. Binary serialization is faster to save and load, but may not be possible to load on a different operating system.
 
+OPTIONS_DB_SERVER_BINARY_SERIALIZATION
+Allows server to use Binary serialization for client-server interaction. Binary serialization is faster to transfer, but may not be possible to read message on a different operating system and architectures.
```

Comment:
Allows -> Forces
...but may not be compatible between different operating systems and...

Reply:
This is still misleading, as the game checks the version strings before using binary serialization, and if they don't match, reverts to XML, so when will this be relevant for different OS or versions?

If you want an option to force XML, that's fine, but phrase it as that, and don't claim it's helpful for cases where it isn't (shouldn't?) be...

Reply:
Tried to rewrite it.